v21.1/take-full-and-incremental-backups.md

```diff
@@ -107,25 +111,105 @@ If it's ever necessary, you can then use the [`RESTORE`][restore] command to res
 
 {% include_cached copy-clipboard.html %}
 ~~~ sql
-> RESTORE FROM '{subdirectory}' IN '{destination}';
+> RESTORE FROM '{subdirectory}' IN '{collection-location}';
 ~~~
 
 {{site.data.alerts.callout_info}}
 <span class="version-tag">New in v21.1:</span> `RESTORE` will re-validate [indexes](indexes.html) when [incremental backups](take-full-and-incremental-backups.html) are created from an older version (v20.2.2 and earlier or v20.1.4 and earlier), but restored by a newer version (v21.1.0+). These earlier releases may have included incomplete data for indexes that were in the process of being created.
 {{site.data.alerts.end}}
 
+## Backup collections
+
+When running a [full backup](#full-backups) to a specified storage location, a **backup collection** will be created in that storage location. A backup collection can contain full backups and subsequent [incremental backups](#incremental-backups). (If a full backup is not present in a collection when an incremental backup is run, then a full backup will be taken.) A collection can hold multiple backup "chains", which consist of a full backup and its incremental backups.
```

Comment:
I get it—and I realize we haven't in the past used "chain" to document full backups + incremental backups in the public docs. 

My feeling was that it's a good way to distinguish between a collection (multiple backups in a location) and then the full & incrementals on an _individual_ basis. What I hadn't thought about is if users do just think about a full backup + its incremental as a "whole" or one backup. I found it easier to think about them as chains, maybe because it makes understanding the dir structure easier. @livlobo what do you think? I can definitely rework to omit "chain" if we don't want to introduce that.

An ideal (but maybe not realistic?) would be to define terminology for the different intersections of a backup at some point. It could make the docs more user-friendly consistency wise.

Reply:
I like the use of the word "chain" as it helps describe the physical representation of backups in various directories (e.g. the examples below).  

We could have an "advanced section" in the docs that contains the examples that illustrate the directory structure and uses the word chain. Everywhere else in the docs, we use David's phrasing. Idk, maybe this complicates things.